Bug fix: need to test if FORCE_REVIEW_SCREEN is declared

andywerner opened the pull request.

Commit:
Bug fix: need to test if FORCE_REVIEW_SCREEN is declared

ghost commented:
<img src="https://www.deepcode.ai/icons/green_check.svg" width= "50px" align= "left"/> Congratulations :tada:. DeepCode [analyzed](https://www.deepcode.ai/app/gh/mrbeam/MrBeamPlugin/40d6c7e8dd6218f684c564c65695193ec9e253ab/mrbeam/MrBeamPlugin/36e33e01b8694628c8116c20b1d00a9356668b26/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) your code in 7.59 seconds and we found no issues. Enjoy a moment of no bugs :sunny:.

#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/mrbeam/MrBeamPlugin/40d6c7e8dd6218f684c564c65695193ec9e253ab/mrbeam/MrBeamPlugin/36e33e01b8694628c8116c20b1d00a9356668b26/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=mrbeam)

Commit:
Bug fix: Only ask new users.
Re-introducing the 'ask' setting and adding new "doNotAskAgain" setting
Also adding more metadata to usage handler.

khaledsherkawi committed:
closing button fixed
